Jenkinsfile_CNP

```diff
@@ -6,7 +6,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "nodejs"
 def product = "civil"
 def component = "ccd"
-def camundaBranch = "master"
```

Comment:
Marking as "Request changes" to prevent this from being merged in.

This also seem to highlight a dependency on CIV-6580, so this ticket cannot be approved until CIV-6580 is past PO Approval.